HW20/roles/hw202/tasks/install.yml

```diff
@@ -0,0 +1,12 @@
+- name: Install a .deb package
+  become: true
+  apt:
```

Comment:
use full path like you have for most modules here. `ansible.builtin.file:` for example